includes/Modules/Analytics_4/Report/Filters/Filter.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics_4\Report\Filter
+ *
+ * @package   Google\Site_Kit\Modules\Analytics_4\Report\Filters
+ * @copyright 2023 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Analytics_4\Report\Filters;
+
+use Google\Site_Kit_Dependencies\Google\Service\AnalyticsData\FilterExpression as Google_Service_AnalyticsData_FilterExpression;
+
+/**
+ * Interface for a filter class.
+ *
+ * @since 1.106.0
```

Comment:
Let's add another `@since` to help track the file move. I'll apply this myself to save the round

```suggestion
 * @since 1.106.0
 * @since n.e.x.t Moved from `Analytics_4/Report/Dimension_Filters` to `Analytics_4/Report/Filters`  for use with both dimensions and metrics.
```